docs/products/compute/compute-instances/guides/metadata/index.md

```diff
@@ -0,0 +1,218 @@
+
+    User data can be submitted directly in the Cloud Manager, Linode CLI, or Linode API. It's also often programmatically provided through IaC (Infrastructure as Code) provisioning tools like [Terraform](/docs/guides/how-to-build-your-infrastructure-using-terraform-and-linode/).
+
+When a Compute Instance first boots up, cloud-init runs locally on the system, accesses the metadata, and then configures your system using that metadata.
+
+## Comparison to StackScripts
+
+Similar to Metadata, Linode's [StackScripts](/docs/products/tools/stackscripts/) service can be used to automate system provisioning. However, the StackScripts service is Linode-specific and does not interface with cloud-init. If you wish to keep your system provisioning tools cloud-agnostic and industry-standard, we recommend using the Metadata service.
+
+## Availability
+
+The Metadata service is currently in a limited beta. As such, only users enrolled in the beta are able to use the service. Additionally, the service is limited to the following distribution images and data centers:
+
+- **Data centers:** Washington, DC (`us-iad`) and Paris (`fr-par`)
+- **Distributions:** Ubuntu 22.04 LTS and Ubuntu 20.04 LTS
+
+## Add User Data When Deploying a Compute Instance {#add-user-data}
+
+The Metadata service is always active, so there's no need to enable it. User data can be provided to the Metadata service, which is then consumed by cloud-init when your Compute Instance boots up for the first time.
+
+{{< tabs >}}
+{{< tab "Cloud Manager" >}}
+1.  Navigate to the **Linodes** page in the [Cloud Manager](http://cloud.linode.com) and click the **Create Linode** button. This opens the **Create Linode** form.
+
+1.  Fill out the form with your desired settings. Be sure to select one of the supported distribution images and data centers.
+
+1.  Expand the *Add User Data* section and enter your user data into the **User Data** field.
+
+    ![Screenshot of the Add User Data section in the Cloud Manager](user-data-section.png)
+
+    If you are unfamiliar with cloud-init, you can review the [Cloud-Config Usage and Examples](/docs/products/compute/compute-instances/guides/metadata-cloud-config/) guide for help creating a cloud-config file.
+
+1.  Once you are ready, click the **Create Linode** button to deploy the instance.
+{{< /tab >}}
+{{< tab "Linode CLI" >}}
+```command
+linode-cli linodes create \
+  --label new-instance-with-metadata \
+  --region us-iad \
+  --type g6-standard-2 \
+  --image linode/ubuntu22.04 \
+  --root_pass [your-root-password] \
+  --metdata.user_data [your-user-data]
+```
+
+Replace *[your-root-password]* with a strong root password and *[your-user-data]* with the Cloud-config data or script you wish to use. When using the API or CLI, user data must be a [base64-encoded string](https://en.wikipedia.org/wiki/Base64). You can output your Cloud-config or script file as base64 by running the following command:
```

Comment:
@mwildman In some places `Cloud-config` is used (not counting the start of a sentence), and in others `cloud-config` is used. This seemed like it could be standardized to `cloud-config`, which appears to what the official docs do, but I didn't make this change

Reply:
Donezo